go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/gosimple/slug v1.9.0
 	github.com/grafana/grafana-aws-sdk v0.4.0
-	github.com/grafana/grafana-plugin-sdk-go v0.104.0
+	github.com/grafana/grafana-plugin-sdk-go v0.104.1-0.20210610153358-11c9baf0fd86
```

Comment:
I'm temporarily vendoring https://github.com/grafana/grafana-plugin-sdk-go/pull/370 to make the CI build for this PR to pass. Once that is merged, I'll update the vendor with the right version.

Reply:
Sure that works. We normally recommend to use a replace directive instead when working towards branches, e.g. `go mod edit -replace=github.com/grafana/grafana-plugin-sdk-go=github.com/grafana/grafana-plugin-sdk-go@<branch commit>`

Reply:
This was addressed in https://github.com/grafana/grafana/pull/35520/commits/782e874c5940d060da11bf790d50f95de3c4b096